coordinator/src/main/java/com/tencent/rss/coordinator/SimpleClusterManager.java

```diff
@@ -72,14 +72,18 @@ public SimpleClusterManager(CoordinatorConf conf) {
     }
   }
 
-  private void nodesCheck() {
+  @VisibleForTesting
+  protected void nodesCheck() {
```

Comment:
I prefer use `default` accessor to `protected`. Some spark code use `default` accessor to provide